src/storage/MonitoringStore.ts

```diff
@@ -55,12 +56,16 @@ export class MonitoringStore<T extends ResourceStore = ResourceStore>
   private emitChanged(changes: ChangeMap): ChangeMap {
     for (const [ identifier, metadata ] of changes) {
       const activity = metadata.get(SOLID_AS.terms.Activity);
-      this.emit('changed', identifier, activity);
-      if (activity && emittedActivities.has(activity.value)) {
```

Comment:
Not completely sure anymore but I think this was a way to get all the typings to line up correctly. Will recheck.

Reply:
The reason is the type guard of `isKnownActivity`. If that function passes we additionally get the typing information that the input activity is one of the three supported activities. `Set.has` only returns a boolean so we would still have to cast in the calls below since the compiler doesn't know the activity is one of the supported types.

Additionally, the reason it is now an array is because then we have the type guard `term is typeof emittedActivities[number]`. Since the set contains strings we can't use it for that purpose. I could introduce a new utility type that extracts the type of a Set entry and then we could have as type guard `term is NamedNode<SetEntry<typeof knownActivities>>` using the original Set and use the set internally in that function.

No strong opinion either way as the difference seems negligible to me.

Reply:
The suggested change would not work btw since the typings of the `has` function require the input type to match the type of the Set entries.

Reply:
…yeah but the `Set` would then just be reverted to be stings.

But it's fine with me, let's just be consistent in naming known/emitted activity.